crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -186,13 +190,32 @@ impl ModuleLoader {
             idx,
             SymbolRefDbForModule::new(Scoping::default(), idx, ScopeId::new(0)),
           );
-          let symbol_ref = self.symbol_ref_db.create_facade_root_symbol_ref(
-            idx,
-            legitimize_identifier_name(resolved_id.id.as_str()).as_ref(),
-          );
 
-          let ext =
-            ExternalModule::new(idx, resolved_id.id, external_module_side_effects, symbol_ref);
+          let renormalize_render_path = !matches!(resolved_id.external, ResolvedExternal::Absolute)
```

Comment:
A better naming would be more readable, https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members?redirectedfrom=MSDN#names-of-properties, 

NIT: `need_renormalize_render_path` maybe better

Reply:
Look good for me.